corehq/apps/users/views/__init__.py

```diff
@@ -566,13 +568,20 @@ def page_context(self):
                 "Any users assigned to roles that are restricted in data access "
                 "by organization can no longer access this project.  Please "
                 "update the existing roles."))
+        if (
```

Comment:
Can you say more about this logic? Making feature flags behave differently in different server environments is a treacherous path to go down. Are you concerned about preserving the existing behavior for other projects using this flag?

Reply:
This was introduced to test out things on icds-staging to ensure that it does not change behavior in other places.

I agree with you that it is not a good way to go forward. It can act as a short term fix that keeps things in order, then we can either create a new setting or replace the previous one completely which I guess would be a much appropriate fix.